### ISSUE TITLE
Answering a VoIP call with no microphone create confusion on the call sender side

### ISSUE THREAD

**Issue by gsouquet** [vector-im/element-web#21315]
### Steps to reproduce

1. Create a DM with a user that has no microphone, or where their computer does not recognise their microphone
2. Call that user
3. Have that user answer the VoIP call

#### What did you expect?

For the caller to be notified that the call could not go through

#### What happened instead?

The callee has a dialog that says "no microphone detected" and the call is ended on their side. On the caller's side, the call keeps on ringing, and they are never notified that the call did not go through

### Will you send logs?

No

**Comment by SimonBrandner** [comment]
This is supposed to give you a chance to answer the call on a different device though I suppose it only confuses people

**Comment by gsouquet** [comment]
Right, the person that was calling me had only once active session, so it would not have been possible for them to answer the call from a different device
